hooks/96-auto-manage-uefi-entries.install

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+check_if_uefi_entry_exists () {
+	for entry in $(efibootmgr); do
+
+		if [[ "$entry" == *"($efi_path"* ]] && [[ "$entry" == *"$partition_uuid"* ]]; then
+			
+			return 1
+		fi
+			
+	done
+	return 0
+}
+
+add_uefi_entry () (
+	for efi in $partition_efis; do
+		
+		# Prepare path which will be inserted into efibootmgr
+		efi_path=$(echo "${efi//$partition_mount}" | sed 's/\//\\/g')
+
+		# Add entry if it doesn't exist
+		if check_if_uefi_entry_exists; then
+
+			echo "Adding UEFI entry for $efi_path found on $partition..."
+			efibootmgr --create --disk /dev/$partition --label "Gentoo $partition" --loader "$efi_path" -u $kenrel_commands &>/dev/null
+		
+		else
+		
+			echo "UEFI Entry already exists for $efi_path on $partition !"	
+		
+		fi
+	done
+)
+
+main () {
+	efi_parttype="c12a7328-f81f-11d2-ba4b-00a0c93ec93b"
+	kernel_commands='crypt_root=UUID=dcb0cc6f-464c-4e98-b91c-e59e3655d161 root=/dev/mapper/gentoo-root rootfstype=ext4 resume=/dev/mapper/gentoo-swap dolvm quiet'
```

Comment:
Here we can borrow the logic from `/usr/lib/kernel/install.d/90-loadentry.install:

```
if [ -n "$KERNEL_INSTALL_CONF_ROOT" ]; then
    if [ -f "$KERNEL_INSTALL_CONF_ROOT/cmdline" ]; then
        BOOT_OPTIONS="$(tr -s "$IFS" ' ' <"$KERNEL_INSTALL_CONF_ROOT/cmdline")"
    fi
elif [ -f /etc/kernel/cmdline ]; then
     BOOT_OPTIONS="$(tr -s "$IFS" ' ' </etc/kernel/cmdline)"
elif [ -f /usr/lib/kernel/cmdline ]; then
     BOOT_OPTIONS="$(tr -s "$IFS" ' ' </usr/lib/kernel/cmdline)"
else
     BOOT_OPTIONS="$(tr -s "$IFS" '\n' </proc/cmdline | grep -ve '^BOOT_IMAGE=' -e '^initrd=' | tr '\n' ' ')"
fi
```

Reply:
Good idea, thx, will use it.